### DISCUSSION TITLE
Swift - casting to date not working as expected

### DISCUSSION THREAD

**Discussion by pkasson** [duckdb/duckdb#14519] [Q&A]
### What happens?

The expectation of the following code should result in a date, as it is in the CSV, which is mm/dd/yyyy.  Instead, it shows as Date(days: 19970)

'''
   case .date:
    TabularData.Column(self.cast(to: Date.self)).eraseToAnyColumn()
'''

The cast is not working - how do you fix this ?

### To Reproduce

Unknown

### OS:

MacOS 14.4.1

### DuckDB Version:

0.10.3

### DuckDB Client:

Swift

### Hardware:

Macbook

### Full Name:

0.10.3

### Affiliation:

None

### What is the latest build you tested with? If possible, we recommend testing with the latest nightly build.

I have tested with a stable release

### Did you include all relevant data sets for reproducing the issue?

Yes

### Did you include all code required to reproduce the issue?

- [X] Yes, I have

### Did you include all relevant configuration (e.g., CPU architecture, Python version, Linux distribution) to reproduce the issue?

- [X] Yes, I have

**Comment by szarnyasg** [comment]
Hi @pkasson. Thanks for reporting this. Unfortunately the Swift client is [not an officially supported client](https://duckdblabs.com/news/2023/10/02/support-policy). I'll leave this open in case someone from the community knows the answer.

**Reply by pkasson** [reply]
Thanks.  Is there a GitHub project where this should be posted instead ?

**Comment by willtemperley** [comment]
The OP already had their question answered here:

https://github.com/duckdb/duckdb-swift/issues/5
